### DISCUSSION TITLE
Subtle Image Glass Icon & Hidden App Title for Win10 immersion

### DISCUSSION THREAD

**Discussion by borovez** [d2phap/ImageGlass#759] [General] [answered]
Hello!

I know you may not want the option to remove the icon/app title toolbar from the app for marketing purposes... but I thought I would suggest it anyways. Maybe just replace it with a small faint watermark in the bottom left hand corner?

Current:
![image](https://user-images.githubusercontent.com/1531030/80397321-f0301e00-8883-11ea-9c02-c8b3b06a90a6.png)

Suggested Example:
![Subtle Logo](https://user-images.githubusercontent.com/1531030/80398360-7ac54d00-8885-11ea-9214-66169e4cf357.jpg)

The reason is, I love this to be a true replacement for the Windows 10 Photos App, but the icon/app name in the title bar is ruining my immersion while using the app with the Windows 10 theme for [ImageGlass](https://imageglass.org/theme/windows-10-dark-27)

**Comment by d2phap** [comment]
Maybe I should allow option in the theme to change the window title bar icon.
And the hiding text on title bar only can be done after this https://github.com/d2phap/ImageGlass/issues/658 closed.

**Comment by d2phap** [comment, marked as answer]
@borovez 
This will be possible in the upcoming release v8.0, while
- Customizing IG logo can be done in theme pack, you can even hide it as well (full details will be updated at https://github.com/ImageGlass/theme); and
- Hiding image info on Title can be done in Settings, currently it's experimental feature (no UI yet), need to directly change in `igconfig.xml` file. See #915

**Reply by baibai92** [reply]
hi,developers.Hiding image info on Title can be done in Settings, this is very good.But not so perfect, I have an idea.I want the image information to be fully displayed, partially hidden, and completely hidden.When the ui is developed, it will provide such a feature.

**Reply by d2phap** [reply]
What do you mean "partially hidden"?

**Reply by baibai92** [reply]
Resolution, Size, Modification Time, Scaling.This information can be optionally hidden.
